tests/integration/targets/aci_vrf_fallback_route_group/tasks/main.yml

```diff
@@ -0,0 +1,376 @@
+- name: Execute tasks only for ACI v5+ and non-cloud sites
+  when: 
+  - query_cloud.current == [] # This condition will execute only non-cloud sites
+  - version.current.0.topSystem.attributes.version is version('5', '>=')
+  block:  # block specifies execution of tasks within, based on conditions
+    - name: Clear existing setup
+      cisco.aci.aci_tenant:
+        <<: *aci_info
+        tenant: ansible_test
+        state: absent
+
+    - name: Ensure tenant exists for tests to kick off
+      cisco.aci.aci_tenant: &aci_tenant_present
+        <<: *aci_info
+        state: present
+        tenant: ansible_test
+      register: tenant_present
+
+    - name: Create vrf
+      cisco.aci.aci_vrf:
+        <<: *aci_tenant_present
+        vrf: vrf_test
+
+    - name: Create vrf Fallback Route - check mode
+      cisco.aci.aci_vrf_fallback_route_group: &fbrg_info
+        <<: *aci_info
+        tenant: ansible_test
+        vrf: vrf_test
+        name: test_fallback_route_group
+        fallback_route: 1.1.1.1/2
+        fallback_members: 
+          - 1.1.1.1
+          - 2.2.2.2
+          - 4.4.4.4
+        description: Test for Fallback Route Group
+        state: present
+      check_mode: true
+      register: cm_create_vrf_fbrg
+
+    - name: Create vrf Fallback Route - Normal mode
+      cisco.aci.aci_vrf_fallback_route_group:
+        <<: *fbrg_info
+      register: nm_create_vrf_fbrg
+
+    - name: Verify that vrf_fbrg has been created correctly with the correct attribute values
+      ansible.builtin.assert:
+        that:
+          - cm_create_vrf_fbrg is changed
+          - cm_create_vrf_fbrg.previous == []
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.attributes.descr == "Test for Fallback Route Group"
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.children | to_json | regex_search('"rnhAddr":\s*"1\.1\.1\.1"') is not none
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.children | to_json | regex_search('"rnhAddr":\s*"2\.2\.2\.2"') is not none
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.children | to_json | regex_search('"rnhAddr":\s*"4\.4\.4\.4"') is not none
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.children | to_json | regex_search('"fbrPrefix":\s*"1\.1\.1\.1/2"') is not none
+          - cm_create_vrf_fbrg.proposed.fvFBRGroup.children | length == 4
+          - nm_create_vrf_fbrg is changed
+          - nm_create_vrf_fbrg.previous == []
+          - nm_create_vrf_fbrg.current.0.fvFBRGroup.attributes.name == "test_fallback_route_group"
+          - nm_create_vrf_fbrg.current.0.fvFBRGroup.attributes.dn == "uni/tn-ansible_test/ctx-vrf_test/fbrg-test_fallback_route_group"
+          - nm_create_vrf_fbrg.current.0.fvFBRGroup.attributes.descr == "Test for Fallback Route Group"
+          - nm_create_vrf_fbrg.current.0.fvFBRGroup.children | to_json | regex_search('"rnhAddr":\s*"1\.1\.1\.1"') is not none
```

Comment:
Can we do something like this to avoid multiple use of to_json for all the children?
```
- name: Convert to json
          ansible.builtin.set_fact:
            converted_json: "{{ vrf_fbrg.current[0].fvFBRGroup.children | to_json }}"
            
  Then test the following:
  converted_json | regex_search('"fbrPrefix":\s*"1\.1\.1\.1/2"') is not none
  converted_json | regex_search('"rnhAddr":\s*"4\.4\.4\.4"') is not none
```